tests/unit/cli/cli-test.js

```diff
@@ -79,10 +79,6 @@ afterEach(function() {
   commands = argv = ui = undefined;
 });
 
-function assertVersion(string, message) {
```

Comment:
I don't think the version output is ever tested anymore after removing this, because I expected to see 'version: ' => 'ember-cli: ' in the diff.